addons/schedule/common/src/test/java/org/commonjava/indy/schedule/ScheduleTest.java

```diff
@@ -28,6 +28,7 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.Ignore;
 
 import java.util.Collection;
 
```

Comment:
So maybe @Ignore at class level is more straightforward?

Reply:
That's right, ;-) Will change it.